peripherals/cordic/cordic_g4.yaml

```
@@ -0,0 +1,50 @@
+# CORDIC coprocessor provides hardware acceleration
+# for computing math funcitons
+
+CORDIC:
+  CSR:
+    RRDY:
+      _read:
+        NotReady: [0, "Results from computation are not read"]
+        Ready: [1, "Results are ready, this flag will be automatically cleared once value is read"]
+      _write:
+        Clear: [0, "Clear the computation results"]
+    ARGSIZE:
+      Size32: [0, "Use 32 bit input values"]
+      Size16: [1, "Use 16 bit input values"]
+    RESSIZE:
+      Size32: [0, "Use 32 bit output values"]
+      Size16: [1, "Use 16 bit output values"]
```

```suggestion
      Bits32: [0, "Use 32 bit input values"]
      Bits16: [1, "Use 16 bit input values"]
    RESSIZE:
      Bits32: [0, "Use 32 bit output values"]
      Bits16: [1, "Use 16 bit output values"]
```

We've usually used `BitsN` for this sort of enum, what do you think?

---

Sounds good!